CMakeLists.txt

```diff
@@ -302,7 +302,7 @@ function(strip_symbols targetName outputFilename)
           TARGET ${targetName}
           POST_BUILD
           VERBATIM 
-          COMMAND ${OBJCOPY} --only-keep-debug ${strip_source_file} ${strip_destination_file}
+          COMMAND ${OBJCOPY} ${strip_source_file} ${strip_destination_file}
```

Comment:
Removing --only-keep-debug turns the objcopy into just a copy and not what I originally intended which is to copy just the symbols and source/line info.  Why are we doing this?